test/WebJobs.Script.Tests/Workers/MemoryMappedFileAccessorTests.cs

```diff
@@ -17,22 +18,45 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Workers
     /// </summary>
     public class MemoryMappedFileAccessorTests
     {
+        private readonly IEnvironment _testEnvironment;
         private readonly IMemoryMappedFileAccessor _mapAccessor;
 
         public MemoryMappedFileAccessorTests()
         {
             ILogger<MemoryMappedFileAccessor> logger = NullLogger<MemoryMappedFileAccessor>.Instance;
+            _testEnvironment = new TestEnvironment();
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
                 _mapAccessor = new MemoryMappedFileAccessorWindows(logger);
             }
             else
             {
-                _mapAccessor = new MemoryMappedFileAccessorUnix(logger);
+                _mapAccessor = new MemoryMappedFileAccessorUnix(logger, _testEnvironment);
             }
         }
 
+        [Fact]
+        public void Use_AppSetting_Directory_Unix()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
There's no cleaner way to ignore on Windows?

Reply:
I was disappointed to see that people are recommending doing [this](https://stackoverflow.com/a/4421941/3132415). I went with keeping it simple, but I agree, it's not ideal.

Reply:
Just checking - isn't there a skip option for the test?